server/views/pages/curfewAddress/addResidentDetails.njk

```diff
@@ -22,7 +22,7 @@
                 <h2 class=class="govuk-summary-card__title">Preferred address</h2>
                 <ul class="govuk-summary-card__actions">
                     <li class="govuk-summary-card__action">
-                        <a class="govuk-link govuk-link--no-visited-state" href="prison/assessment/{{ assessmentSummary.prisonNumber }}/curfew-address/find-address">Change<span class="govuk-visually-hidden"> preferred address</span></a>
+                        <a class="govuk-link govuk-link--no-visited-state" href="/prison/assessment/{{ assessmentSummary.prisonNumber }}/curfew-address/find-address">Change<span class="govuk-visually-hidden"> preferred address</span></a>
```

Comment:
**issue:** we shouldn't hard code paths in anywhere. can we use to path syntax instead?